spec/openapi.yaml

```diff
@@ -426,6 +426,44 @@ paths:
         503:
           $ref: "#/components/responses/ServiceUnavailable"
 
+  /api/v1/update/{env}:
+    parameters:
+      - in: path
+        name: storage
+        description: The target environment (python or system) for update
+        required: true
+        schema:
+          type: string
+          enum: ["python", "system"]
+    get:
+      summary: Retrieve information about available updates of packages and their versions
+      responses:
+        200:
+          description: OK
+          headers:
+            Update-Available:
+              schema:
+                type: bool
+                description: Whether there's a package update available
+                example: true
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/PackagesUpdateInfo"
```

Comment:
Please use `OnOff` with that one schema, and rename PackageUpdateInfo to PythonPackageUpdateInfo